community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/slurmsync.py

```diff
@@ -68,12 +68,23 @@ def apply(self, nodes:List[str]) -> None:
         run(f"{lookup().scontrol} update nodename={hostlist} state=power_up")
 
 @dataclass(frozen=True)
-class NodeActionIdle():
+class NodeActionResume():
     def apply(self, nodes:List[str]) -> None:
         hostlist = util.to_hostlist(nodes)
         log.info(f"{len(nodes)} nodes to idle ({hostlist})")
         run(f"{lookup().scontrol} update nodename={hostlist} state=resume")
 
+
+# Difference between this and resume you may ask?
+# Resume adds powerup flags to node state which is not
+# compatible with some node state transitions (namely POWER(ING/ED) OFF)
+@dataclass(frozen=True)
+class NodeActionIdle():
```

Comment:
nice

Reply:
From what I see, you replaced all  usages `NodeActionIdle` with `NodeActionResume `, right?
Shall we just remove `NodeActionIdle` then?